### PR TITLE
EWPP-0000: Translations guide.

### PR DESCRIPTION
**Pull request by upchuk**

### CODE REVIEW

File: GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+# Translations v2 User Guide
```
Comment (hernani) [inline GUIDE.md]: change to USER_GUIDE.md

File: GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+The v2 of the translation system is a floor-to-ceiling overhaul of how you manage translations
```
Comment (hernani) [inline GUIDE.md]: how you manage content translations.

File: GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+of content (Node). It no longer uses TMGMT and is organised around the concept of a Translation Request (used both for local and remote translations).
+
+Although this system can be used on other entity types as well, this guide will be using Node as an example.
+
+## Translation Dashboard
+
+The entry point of the new system is the content translation dashboard, located on the `Translate` tab of the Node.
+
+On this page you can see:
```
Comment (hernani) [inline GUIDE.md]: In this page you can find:

File: GUIDE.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+* A list of the existing Node translations
+* The ongoing local translations (the ones that have been started already but not yet finished (_synchronised_))
+* The ongoing remote (ePoetry) translations
+
+The dashboard also has a submenu from where you can access the individual dashboards of the local and remote translations.
+
+## Access
+
+Access to translate is restricted to the Translator role.
```
Comment (hernani) [inline GUIDE.md]: to "translate" tab

File: GUIDE.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+       -> set by ePoetry to indicate that they cancelled this translation. The flow for this translation stops here.
+     * **Suspended** [in ePoetry]
+       -> set by ePoetry to indicate that they suspended this translation. They can unsuspend later.
+
+#### Extra languages
+
+Once a translation request has been accepted by ePoetry, you have the option to add extra languages to the request if you haven't added all of them yet. On the translation request page, you have an `Add new languages` button that will take you to a form
+where you can choose the other languages. Note that you cannot remove existing languages from the request.
+
+The added languages then go through the same flow in ePoetry like the original ones: they get marked as Accepted, Ongoing etc.
```
Comment (hernani) [inline GUIDE.md]: Ongoing, etc

File: GUIDE.md [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+
+Note: if the request has Auto-accept or Auto-sync turned on, the above steps would happen automatically as the translation arrives from ePoetry.
+
+#### Failed requests
+
+If the translation request fails and there is an error shown, you need to contact the site administrator. The request is currently marked as Failed and the Log Messages on the request page should
+indicate the error. Once the issue was addressed, the failed request must be marked as Failed & Finished. You can do so by going to the translations
+dashboard and using the `Mark as finished` operation on the respective translation request. This will unblock the flow and a request can be tried again.
+
+### Poetry legacy
```
Comment (hernani) [inline GUIDE.md]: Indicate module needs to be enabled? As we are in the context of the component.

File: GUIDE.md [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+* The ongoing local translations (the ones that have been started already but not yet finished (_synchronised_))
+* The ongoing remote (ePoetry) translations
+
+The dashboard also has a submenu from where you can access the individual dashboards of the local and remote translations.
+
+## Access
+
+Access to translate is restricted to the Translator role.
+
+## Local translations
```
Comment (kugta) [inline GUIDE.md]: Maybe we could elaborate on the scenario when we have multiple major versions.

File: GUIDE.md [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+
+Local translations allow to translate content manually. From the local translation dashboard, you can use the "New translation" link for each language to create a
+new translation. If the Node already has a translation in a previous version, the translation fields will be pre-filled with the values from the previous version translation.
+
+A local translation can be saved as draft (to be continued later), as accepted (to mark that it's in a ready state) and/or directly synchronised (more info on the concept of _synchronisation_ below).
```
Comment (kugta) [inline GUIDE.md]: I would capitalise/emphasise the statuses to make it clearer that these are actual status names, as we will see below.

File: GUIDE.md [hunk 9]
```diff
@@ -0,0 +1,203 @@
...
+
+## ePoetry translations
+
+### Configuration
+
+The ePoetry configurator is located on the ePoetry Remote Translator configuration page: `/admin/structure/remote-translation-provider/epoetry/edit`.
+
+You **must** have the following configured:
+
+* Default contacts (that can be overridden per translation request): Recipient, Webmaster, Editor
```
Comment (kugta) [inline GUIDE.md]: We could maybe elaborate based on the help text provided for the different contacts.

File: GUIDE.md [hunk 10]
```diff
@@ -0,0 +1,203 @@
...
+
+### Translations
+
+ePoetry translations requests can be started from the Remote translations dashboard.
+
+Request values (the mandatory ones are marked accordingly on the form):
+
+* Languages
+  * You can pick as many as you like
+  * If you didn't include all you needed, once the translation request has been accepted by ePoetry, you can also include others to the request
```
Comment (kugta) [inline GUIDE.md]: accepted by DGT in ePoetry?

File: GUIDE.md [hunk 11]
```diff
@@ -0,0 +1,203 @@
...
+* Auto-accept translations (as translations arrive from ePoetry, they get marked as accepted automatically)
+* Auto-sync translations (as translations arrive from ePoetry, they get synchronised automatically)
+* Deadline (when you would like the translations to be done by)
+  * Careful not to pick a weekend in your deadline
```
Comment (kugta) [inline GUIDE.md]: Be careful...

File: GUIDE.md [hunk 12]
```diff
@@ -0,0 +1,203 @@
...
+* Message (a message to the translators at DGT)
+* Contacts (prefilled from the site-wide configuration but you can alter per request)
+
+Once the translation request has been sent, you need to wait for ePoetry to either Accept it or Reject it. If they Reject it, the system gets unblocked and you can resubmit it. If they Accept it, the flow can continue.
```
Comment (kugta) [inline GUIDE.md]: ...wait for DGT to either Accept...

File: GUIDE.md [hunk 13]
```diff
@@ -0,0 +1,203 @@
...
+
+**NOTE: Until ePoetry either Accepts or Rejects the request, no other request can be made to ePoetry for this Node**.
```
Comment (kugta) [inline GUIDE.md]: DGT

File: GUIDE.md [hunk 14]
```diff
@@ -0,0 +1,203 @@
...
+     -> set when all the individual language translations have arrived from ePoetry and the DGT work is done on this request.
+   * **Finished**
+     -> set when
+     * all the individual language translations have been synchronised onto the content or
+     * the request has been replaced by another one via an Update request (see below) or
+     * the request has failed and has been manually marked as finished
+   * **Failed**
+     -> set when the request has failed due to a network issue, connection issue, exception, etc.
+   * **Failed & Finished**
+     -> manually-set status to unblock a Failed request
```
Comment (kugta) [inline GUIDE.md]: Who can set this where?

File: GUIDE.md [hunk 15]
```diff
@@ -0,0 +1,203 @@
...
+#### Extra languages
+
+Once a translation request has been accepted by ePoetry, you have the option to add extra languages to the request if you haven't added all of them yet. On the translation request page, you have an `Add new languages` button that will take you to a form
+where you can choose the other languages. Note that you cannot remove existing languages from the request.
+
+The added languages then go through the same flow in ePoetry like the original ones: they get marked as Accepted, Ongoing etc.
+
+#### Update requests
+
+Once a translation request has been Accepted by ePoetry AND you make a new version of the content, you can make an `Update` request. This informs ePoetry that the content has changed and
```
Comment (kugta) [inline GUIDE.md]: DGT

File: GUIDE.md [hunk 16]
```diff
@@ -0,0 +1,203 @@
...
+*NOTE: when using the corporate workflow, a new version of the content means that you have created a new Validated revision.*
+
+#### Translation review
+
+When a given language translation is in Review, you can click the Review button for that language and check the translation values. You can then do the following:
+
+* Accept the translation -> saves the request with this translation marked as Accepted
+* Synchronise the translation -> synchronises the translation with the Node
+
+Note: if the request has Auto-accept or Auto-sync turned on, the above steps would happen automatically as the translation arrives from ePoetry.
```
Comment (kugta) [inline GUIDE.md]: Maybe we could explain that auto-sync means also auto accept (if this is true :))

File: GUIDE.md [hunk 17]
```diff
@@ -0,0 +1,203 @@
...
+
+#### Failed requests
+
+If the translation request fails and there is an error shown, you need to contact the site administrator. The request is currently marked as Failed and the Log Messages on the request page should
+indicate the error. Once the issue was addressed, the failed request must be marked as Failed & Finished. You can do so by going to the translations
+dashboard and using the `Mark as finished` operation on the respective translation request. This will unblock the flow and a request can be tried again.
+
+### Poetry legacy
+
+For sites that have used already Poetry, there is a Poetry legacy table for administrators who can check the last Poetry reference that was sent for each node.
```
Comment (kugta) [inline GUIDE.md]: please double check the roles, translator should also have access

File: GUIDE.md [hunk 18]
```diff
@@ -0,0 +1,203 @@
...
+Apart from the individual Node translations dashboard, you also have a dashboard where you can have an overview over all the ePoetry translation requests:
+
+`/admin/content/epoetry-translation-requests`
+
+## Synchronisation
+
+Synchronisation means that the translation values will be saved onto the Node itself (creating or updating the translation).
+
+Each translation request (both local or remote) is started from a specific Node version. This means that when the synchronisation happens, the translation values
+get saved onto that specific revision. This means that if you have ongoing translations and you make a new draft, those translations will be synced onto the previous revision, the new draft
```
Comment (kugta) [inline GUIDE.md]: version instead of revision, no? in case we published the validated in the meantime.

Comment (kugta) [reply GUIDE.md]: OK I see the workflow specific part below.

File: GUIDE.md [hunk 19]
```diff
@@ -0,0 +1,203 @@
...
+
+This is because of the "upstream" impossibility. Any new draft may contain data structure changes that would be impossible to reconcile with the structure of the previous revision. Which means that
+we cannot cleanly save the translations onto the new structure.
+
+An exception to the above rule is made if the site is using the Corporate Workflow. See relevant section.
+
+## Corporate workflow rules
+
+* If the translation is started from a revision in Validated state, the values will be synchronised onto the Published revision of the same version (if one has been made in the meantime).
+* No translation can be started until there is a Validated or Published version (new version) of the Node.
```
Comment (kugta) [inline GUIDE.md]: (new major version) ?

File: GUIDE.md (hunk 7 above)
Comment (upchuk) [reply GUIDE.md]: This is covered below in the corporate workflow rules

File: GUIDE.md (hunk 10 above)
Comment (upchuk) [reply GUIDE.md]: I'm keeping ePoetry everywhere, no reference to DGT..just to not add confusion

File: GUIDE.md (hunk 14 above)
Comment (upchuk) [reply GUIDE.md]: It's explained below

File: GUIDE.md (hunk 16 above)
Comment (upchuk) [reply GUIDE.md]: It's irrelevant. It sets the status to Sync, so accept is not involved anymore..there is no revisions of the request to go through multiple statuses

File: GUIDE.md (hunk 17 above)
Comment (upchuk) [reply GUIDE.md]: They do

File: GUIDE.md (hunk 19 above)
Comment (upchuk) [reply GUIDE.md]: ah right

**Pull request merged**